test/fc_mock.tsv

```diff
@@ -0,0 +1,9 @@
+Donor UUID	Sample UUID	File Type	Specimen UUID	File URLs
+d1	s1	crai	sp1	gs://junk.crai
+d1	s1	cram	sp1	gs://junk.cram
+d1	s2	crai	sp1	s3://junk.crai
+d1	s2	cram	sp1	s3://junk.cram
+d2	s3	crai	sp1	gs://junk.crai
+d2	s3	cram	sp1	gs://junk.cram
+d3	s4	crai	sp1	gs://junk.crai
+d3	s4	cram	sp1	gs://junk.cram
```

Comment:
Could you please add a test case where there are not symmetrical crai/cram? E.g., a sample with only 1 file and a sample with 3 files?

Reply:
done, all described in the code